vital/types/camera_intrinsics.h

```diff
@@ -145,13 +151,26 @@ class VITAL_EXPORT simple_camera_intrinsics :
                            const vector_2d& principal_point,
                            const double aspect_ratio=1.0,
                            const double skew=0.0,
+                           const unsigned int image_width=0,
```

Comment:
I would put these new width and height parameters after `dist_coeffs` to avoid breaking existing uses of this constructor that use distortion coefficients.

Reply:
It's done.